subsys/bluetooth/mesh/shell/shell.c

```diff
@@ -361,33 +448,37 @@ struct bt_mesh_prov bt_mesh_shell_prov = {
 	.static_val = NULL,
 	.static_val_len = 0,
 	.output_size = 6,
-	.output_actions = (BT_MESH_DISPLAY_NUMBER | BT_MESH_DISPLAY_STRING),
+	.output_actions = (BT_MESH_BLINK | BT_MESH_BEEP | BT_MESH_VIBRATE | BT_MESH_DISPLAY_NUMBER |
+			   BT_MESH_DISPLAY_STRING),
 	.output_number = output_number,
 	.output_string = output_string,
 	.input_size = 6,
-	.input_actions = (BT_MESH_ENTER_NUMBER | BT_MESH_ENTER_STRING),
+	.input_actions =
+		(BT_MESH_ENTER_NUMBER | BT_MESH_ENTER_STRING | BT_MESH_TWIST | BT_MESH_PUSH),
 	.input = input,
 	.input_complete = prov_input_complete,
 };
 
 static int cmd_static_oob(const struct shell *sh, size_t argc, char *argv[])
 {
+	struct bt_mesh_prov *prov = (struct bt_mesh_prov *)bt_mesh_prov_get();
+
 	if (argc < 2) {
-		bt_mesh_shell_prov.static_val = NULL;
-		bt_mesh_shell_prov.static_val_len = 0U;
+		prov->static_val = NULL;
+		prov->static_val_len = 0U;
 	} else {
-		bt_mesh_shell_prov.static_val_len = hex2bin(argv[1], strlen(argv[1]),
+		prov->static_val_len = hex2bin(argv[1], strlen(argv[1]),
```

Comment:
Is it required `static_val` to be a static global variable?

Reply:
You mean that we should not provide a way to set/alter the static value?